src/types.rs

```diff
@@ -96,6 +99,16 @@ impl Subscription {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct AppleSubscriptionInfo {
+    /// https://developer.apple.com/documentation/appstoreserverapi/status
+    pub status: u8,
+    /// Whether the subscription will renew automatically
+    pub auto_renew_status: bool,
+    /// Subscription expiration date in seconds since unix epoch
+    pub renewal_date: i64,
```

Comment:
Do we know if this this date is elapsed or a future date if the status is expired?